res/controllers/lib/mididispatcher.mjs

```diff
@@ -58,7 +58,7 @@ export class MidiDispatcher {
      * @param {Array} data - The incoming MIDI data.
      * @param {number} timestamp - The timestamp that Mixxx received the MIDI data at.
      */
-    receiveData(data, timestamp) {
+    receiveMidiData(data, timestamp) {
```

Comment:
Why did you change this? Midi is that already in the class name.

Reply:
https://github.com/mixxxdj/mixxx/pull/2868#issuecomment-649145697

Reply:
Do you have suggestions for another name?

Reply:
Maybe use `handleInput` or something like that for controller modules?

I think it would be beneficial to leave Midi out of the function name so that we can potentially make a HidDispatcher class with a similar API later on.

Reply:
I renamed `receiveData` to `handleInput`. `receiveData` is passive language; `handleInput` makes it clearer that the functions do more.

I renamed `receiveMidiData` in the MidiDispatcher module to `handleMidiInput`. For the HidDispatcher or HidPacketParser or whatever we call it, we can add a function that takes the same `data` and `timestamp` arguments, but I don't think the function names need to be identical. To the contrary, I think it would be helpful to have slightly different function names for each to emphasize whether they handle MIDI or HID.